.bazelversion

```diff
@@ -0,0 +1 @@
+6.4.0
```

Comment:
Note - ci should be configured to use the same version.

Reply:
We should probably add a note here or in the ci config that these versions should be matching.